sopel/bot.py

```diff
@@ -189,6 +190,15 @@ def hostmask(self) -> Optional[str]:
 
         return self.users.get(self.nick).hostmask
 
+    @property
+    def plugins(self) -> Mapping[str, plugins.handlers.AbstractPluginHandler]:
+        """A dict of the bot's currently loaded plugins.
+
+        :return: an immutable map of plugin name to plugin object
+        :rtype: Mapping[str, plugins.handlers.AbstractPluginHandler]
```

Comment:
This line is, indeed, not needed thanks to the signature that is picked up by Sphinx now.

Reply:
Can confirm:

![image](https://user-images.githubusercontent.com/164140/145733706-508ba489-ee56-4665-a21a-9520e81f866d.png)
I love having [unstable docs](https://unstable.docs.sopel.chat/bot.html#sopel.bot.Sopel.plugins) built on every commit.